src/CONST.js

```diff
@@ -3,13 +3,16 @@ const NEW_EXPENSIFY_URL = 'https://new.expensify.com';
 const PLATFORM_OS_MACOS = 'Mac OS';
 
 const CONST = {
+    get ANDROID_PACKAGE_NAME() {
+        return 'com.expensify.chat';
```

Comment:
Can you explain this to me please ? I'm curious why it's not a regular const.

Reply:
Actually was trying to do something [fancy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) here but it broke everything on web 😄

Reply:
Basically a getter let's you access `this.someObjectProperty` from within the getter. But it does not work the other way around 🤦